spec/discourse_api/api/categories_spec.rb

```diff
@@ -182,6 +182,29 @@
     end
   end
 
+  describe "#reorder_categories" do
+    before do
+      stub_post("#{host}/categories/reorder_categories").to_return(
+        body: fixture("notification_success.json"),
+          headers: {
+          content_type: "application/json",
+        },
+      )
+    end
+
+    it "makes a categories reordering request" do
+      payload = {"1": 2, "2": 3, "3": 4, "4": 1}
+      response = subject.reorder_categories(mapping: payload.to_json)
+      expect(
+        a_post("#{host}/categories/reorder").with(
+          body:
+            { mapping: payload.to_json }
```

Comment:
```suggestion
            "mapping=#{CGI.escape(payload.to_json.to_s)}"
```

Reply:
We have Faraday configured to send url_encoded requests. Not really sure why, just the way it has been.